vissl/data/ssl_transforms/img_pil_to_rgb_mode.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+from typing import Any, Dict, List, Union
+
+from PIL import Image
+from classy_vision.dataset.transforms import register_transform
+from classy_vision.dataset.transforms.classy_transform import ClassyTransform
+
+
+@register_transform("ImgPil2RGB")
+class ImgPil2RGB(ClassyTransform):
```

Comment:
we should modify these name accordingly after we settle on the file name :)

Reply:
It should now be fixed in 1d0d4a1162346fde86938fa5f9e53291ef55f55e